lib/config/eazy_yml.go

```diff
@@ -7,6 +7,7 @@ import (
 // EazyYml stuct for eazy.yml file in the repo's
 type EazyYml struct {
 	Name        string
+	Runtime     string `yaml:"runtime"`
```

Comment:
I'm unsure if this should live here, since I assume child dependencies will also have their own ci config?

Reply:
I'm leaning towards a command line option or .eazy/config type of thing, because *hopefully* no one actually deploys with runc or docker with this in production, so it's really the runtime they want to test locally on their machine on or on a Jenkins box. It's really what is convenient for users. Did you have an opinion on the matter because this opinion isn't really strong?

Reply:
I was also thinking cli flag would be the way to go. Maybe also throw in an env variable just incase people want to set it in a repeatable way?